src/api/procedures/reserveTicker.ts

```diff
@@ -47,6 +47,13 @@ export async function prepareReserveTicker(
   } = this;
   const { ticker, extendPeriod = false } = args;
 
+  if (!isPrintableAscii(ticker)) {
+    throw new PolymeshError({
+      code: ErrorCode.ValidationError,
+      message: 'Only printable ASCII is alowed as ticker name',
+    });
+  }
```

Comment:
Both this check and the next one are unnecessary, since those validations are done in `stringToTicker`

Also, please make `stringToTicker` also validate that the string is not empty (change the error message to `Ticker length must be between 1 and ${MAX_TICKER_LENGTH} characters`, and validate that the string is all upper case (separate error)